### PR TITLE
add kubearmor to projects list

### PR DESCRIPTION
**Pull request by nam-jaehyun**
add the KubeArmor project on the projects list
- https://github.com/accuknox/KubeArmor

### CODE REVIEW

File: src/pages/projects.js [hunk 1]
```diff
@@ -544,6 +544,37 @@ const ProjectDescriptions = () => (
...
+	    <a href="https://kubearmor.com">
+	      <b>Website</b>
+	    </a>{" "}
+	    |{" "}
+            <a href="https://github.com/accuknox/KubeArmor">
+              <b>GitHub</b>
+            </a>{" "}
+          </p>
+          <p>
+	  KubeArmor a container-aware runtime security enforcement system that
```
Comment (qmonnet) [inline src/pages/projects.js:569]: The verb is missing in clause.

Comment (qmonnet) [reply src/pages/projects.js:569]: (Note also that the projects were sorted in alphabetical order so far, this is not the case for this addition.)

Comment (nam-jaehyun) [reply src/pages/projects.js]: Thanks for the comments.
I added the missing verb ("is") and sorted the projects in alphabetical order.

Comment (nam-jaehyun) [reply src/pages/projects.js]: Is there anything that I need to do for the merging process?

**Review by lmb — APPROVED** [review]

**Comment by dthaler** [comment]
Merging per BSC meeting 2021-11-10

**Pull request merged**